package.json

```diff
@@ -23,7 +23,7 @@
     "publish-extension": "tfx extension publish --manifest-globs azure-devops-extension.json src/Samples/**/*.json"
   },
   "dependencies": {
-    "azure-devops-extension-api": "~1.150.2",
+    "azure-devops-extension-api": "^1.150.3",
```

Comment:
Why was this change made? We need to lock to a specific "minor" version like "150". We typically release the API package once a deployment starts going out, but don't want to update the sample until the deployment is finished on all rings, to ensure the extension will work on any account

Reply:
Do you mean the change from ~ to ^ or from 150.2 to 150.3?

~ to ^ is just what npm did when I updated the version and I will change it back.

150.2 to 150.3 is because the interfaces used by the new sample were not added to the api package yet in 150.2 - that's why I asked for the new version on the email thread earlier today. 
The objects with these interfaces exist server-side on all versions of M150.

Reply:
Update